src/views/Discover.jsx

```diff
@@ -0,0 +1,41 @@
+/* eslint-disable no-shadow */
+/* eslint-disable no-unused-vars */
+import React, { useState, useEffect } from 'react';
+import { Link, useNavigate } from 'react-router-dom';
+import { FaDog } from 'react-icons/fa';
+import axios from 'axios';
+
+import CardStack from '../components/Discover/CardStack';
+
+export default function Discover() {
+  const [users, setUsers] = useState([]);
+
+  function getUsers(user) {
+    console.log('making request');
+    axios.get('https://localhost:3001/api/discover', {
```

Comment:
why are we using full URL here rather that just '/discover' we will want this to not be explicit on deployment